src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/TestHelpers.java

```diff
@@ -42,22 +46,13 @@
     public static final String PARTY_TYPE_CD = "PARTYCD";
     public static final String ROLE_TYPE_CD = "ROLECD";
 
-    public static InitialPackage createInitalPackage(Court court, List<DocumentProperties> documentProperties) {
+    public static InitialPackage createInitalPackage(ca.bc.gov.open.jag.efilingapi.api.model.Court court, List<DocumentProperties> documentProperties) {
```

Comment:
I wonder if we need to start prefixing the models. So don't collisions like this. Although I'm not a fan of it either....

Reply:
as we spoke, we keep it the same for now ;-)